### ISSUE TITLE
MAIL_FROM config value not documented

### ISSUE THREAD

**Issue by carissableker** [marktennyson/flask-mailing#13]
The following settings worked for Flask-Mail:
```python
MAIL_SERVER = 'smtp.mailtrap.io'
MAIL_PORT = 2525
MAIL_USE_SSL = False
MAIL_USE_TLS = True
MAIL_USERNAME = 'xxxxxxxx'
MAIL_PASSWORD = 'xxxxxxxx'
MAIL_DEFAULT_SENDER = 'name@example.com'
```
However in Flask-Mailing. the app has error:

```
Traceback (most recent call last):
  File "/usr/local/bin/flask", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/site-packages/flask/cli.py", line 990, in main
    cli.main(args=sys.argv[1:])
  File "/usr/local/lib/python3.8/site-packages/flask/cli.py", line 596, in main
    return super().main(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/click/decorators.py", line 84, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/flask/cli.py", line 845, in run_command
    app = DispatchingApp(info.load_app, use_eager_loading=eager_loading)
  File "/usr/local/lib/python3.8/site-packages/flask/cli.py", line 321, in __init__
    self._load_unlocked()
  File "/usr/local/lib/python3.8/site-packages/flask/cli.py", line 346, in _load_unlocked
    self._app = rv = self.loader()
  File "/usr/local/lib/python3.8/site-packages/flask/cli.py", line 402, in load_app
    app = locate_app(self, import_name, name)
  File "/usr/local/lib/python3.8/site-packages/flask/cli.py", line 273, in locate_app
    return find_best_app(script_info, module)
  File "/usr/local/lib/python3.8/site-packages/flask/cli.py", line 68, in find_best_app
    app = call_factory(script_info, app_factory)
  File "/usr/local/lib/python3.8/site-packages/flask/cli.py", line 119, in call_factory
    return app_factory(*args, **kwargs)
  File "/usr/src/app/flask_skm/__init__.py", line 122, in create_app
    mail.init_app(app)
  File "/usr/local/lib/python3.8/site-packages/flask_mailing/mail.py", line 63, in init_app
    self.config = ConnectionConfig(
  File "pydantic/env_settings.py", line 36, in pydantic.env_settings.BaseSettings.__init__
  File "pydantic/main.py", line 406, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for ConnectionConfig
MAIL_FROM
  value is not a valid email address (type=value_error.email)
```
`MAIL_FROM` is set here:
https://github.com/marktennyson/flask-mailing/blob/4ae3d67bad2ad338c6b518ff5cbfea3a632a2400/flask_mailing/mail.py#L71

and perhaps would be better as 
```python
     MAIL_FROM = app.config.get("MAIL_FROM", app.config.get("MAIL_DEFAULT_SENDER")),
```
(or add `MAIL_FROM` in the "basic_demo"/documentation.

**Comment by marktennyson** [comment]
Thanks @carissableker to detect this errors. I will fix this and create a new release.

**Comment by carissableker** [comment]
Other config values when porting from Flask-Mail,
MAIL_USE_SSL --> MAIL_SSL
MAIL_USE_TLS -->  MAIL_TLS

(perhaps just a note in docs as well, as I missed this)

**Comment by marktennyson** [comment]
> Other config values when porting from Flask-Mail,
> MAIL_USE_SSL --> MAIL_SSL
> MAIL_USE_TLS --> MAIL_TLS
> 
> (perhaps just a note in docs as well, as I missed this)

Yeah, I will change this to => `MAIL_USE_TLS` and `MAIL_USE_SSL` respectively. 
Anyway, thanks for letting me know about these changes.

**Comment by marktennyson** [comment]
fixed on the new 0.0.5 release.
